aws-ts-eks/README.md

```diff
@@ -0,0 +1,144 @@
+# AWS EKS Cluster
+
+This example deploys an EKS Kubernetes cluster with an EBS-backed StorageClass and deploys the Kubernetes Dashboard
+into the cluster.
+
+## Deploying the App
+
+To deploy your infrastructure, follow the below steps.
+
+### Prerequisites
+
+1. [Install Pulumi](https://pulumi.io/install)
+2. [Install Node.js 8.11.3](https://nodejs.org/en/download/)
+3. [Configure AWS Credentials](https://pulumi.io/install/aws.html)
+
+If you'd like to follow the optional instructions in step 7 in order to deploy a Helm chart into your cluster, you'll
+also need to set up the Helm client:
+
+1. [Install the Helm client binaries](https://docs.helm.sh/using_helm/#installing-helm)
+2. Initialize the Helm client:
+
+    ```bash
+    $ helm init --client-only
+    ```
+
+### Steps
+
+After cloning this repo, from this working directory, run these commands:
+
+1. Install the required Node.js packages:
+
+	```bash
+	$ npm install
+	```
+
+2. Create a new stack, which is an isolated deployment target for this example:
+
+    ```bash
+    $ pulumi stack init
+    ```
+
+3. Set the required configuration variables for this program:
+
+    ```bash
+    $ pulumi config set aws:region us-west-2
+    ```
+
+   We recommend using `us-west-2` to host your EKS cluster as other regions (notably `us-east-1`) may have capacity
+   issues that prevent EKS clusters from creating:
+
+	```
+	Diagnostics:
+	  aws:eks:Cluster: eksCluster
+		error: Plan apply failed: creating urn:pulumi:aws-ts-eks-example::aws-ts-eks::EKSCluster$aws:eks/cluster:Cluster::eksCluster: error creating EKS Cluster (eksCluster-233c968): UnsupportedAvailabilityZoneException: Cannot create cluster 'eksCluster-233c968' because us-east-1a, the targeted availability zone, does not currently have sufficient capacity to support the cluster. Retry and choose from these availability zones: us-east-1b, us-east-1c, us-east-1d
+			status code: 400, request id: 9f031e89-a0b0-11e8-96f8-534c1d26a353
+	```
+
+	Due to limitations in the Pulumi AWS Network abstraction, the only workaround for this issue is to deploy into an
+	entirely different region. https://github.com/pulumi/pulumi-aws-infra/issues/32 tracks adding an explicit list of
+	availability zones to the Network abstraction in order to improve this experience.
```

Comment:
Replace this paragraph with: "We are tracking enabling creation of VPCs limited to specific AZs to unblock this in `us-east-1` https://github.com/pulumi/pulumi-aws-infra/issues/32".